app/code/Magento/CatalogImportExport/Model/Import/Product/CategoryProcessor.php

```diff
@@ -204,4 +221,21 @@ public function getCategoryById($categoryId)
     {
         return isset($this->categoriesCache[$categoryId]) ? $this->categoriesCache[$categoryId] : null;
     }
+
+    /**
+     * Standardize a string by lowercase-ing it
+     *
+     * @param string $string
+     * @return string
+     */
+    protected function standardizeString(string $string)
```

Comment:
Can you remove the "string" scalar type hint, please? :)
This is only possible with PHP 7+ and Magento's minimum supported PHP version is 5.6.*.